include/openssl/pem.h

```diff
@@ -473,6 +474,19 @@ OPENSSL_EXPORT int PEM_write_PKCS8PrivateKey(FILE *fp, const EVP_PKEY *x,
                                              int klen, pem_password_cb *cd,
                                              void *u);
 
+// PEM_read_bio_Parameters is a generic PEM deserialization function that
+// parses the public "parameters" in |bio| and returns a corresponding
+// |EVP_PKEY|. If |*pkey| is non-null, the original |*pkey| is freed and the
+// returned |EVP_PKEY| is also written to |*pkey|. This is only supported with
```

Comment:
I know it's implied from this statement, but it would be good to explicitly say that `*pkey` *must be initialized* (either to NULL or to an allocated value) prior to being passed in. (Passing in an uninitialized pointer would be UB.)

Reply:
Added clarification